src/__tests__/json-completion.spec.ts

```diff
@@ -30,6 +30,137 @@ describe("jsonCompletion", () => {
         info: "",
         template: '"oneOfEg2": ',
       },
+      {
+        detail: "",
+        info: "",
+        label: "oneOfObject",
+        template: '"oneOfObject": ',
+        type: "property",
+      },
+    ]);
+  });
+  it("should include defaults for enum when available", async () => {
+    await expectCompletion('{ "en| }', [
+      {
+        label: "enum1",
+        type: "property",
+        detail: "",
+        info: "an example enum with default bar",
+        // TODO: should this not autocomplete to default "bar"?
```

Comment:
this is what I was referring to with completion of default enum value, this is why i added the if/else block in #48